src/components/NcAvatar/NcAvatar.vue

```diff
@@ -417,6 +420,16 @@ export default {
 			default: 'body',
 		},
 	},
+
+	setup() {
+		const root = useTemplateRef('main')
+		const isDarkTheme = useIsDarkThemeElement(root)
```

Comment:
Is there a need to check specifically `main` and not `body` and thus use shared composable?

If so, I would at least go with `props.menuContainer`, so it inherits it from container

Reply:
> Is there a need to check specifically `main` and not `body` and thus use shared composable?

To avoid misunderstanding, `main` here is template ref key, not `<main>` query selector

Reply:
> how does talk handle this currently?

`<div class="top-bar__wrapper" :data-theme-dark="isInCall">` 🙈